src/modules/governance/FormattedProposalTime.tsx

```diff
@@ -82,18 +87,33 @@ export function FormattedProposalTime({
       </Typography>
     );
   }
-  const canBeExecuted = timestamp > executionTime;
+  const crossChainExecutionTime = l2Execution ? executionTimeWithL2 : executionTime;
+  const canBeExecuted = timestamp > crossChainExecutionTime;
   return (
     <Typography component="span" variant="caption">
       <Typography
         variant="caption"
         color="text.secondary"
         sx={{ display: { xs: 'none', md: 'inline' } }}
       >
-        {canBeExecuted ? <Trans>Expires</Trans> : <Trans>Can be executed</Trans>}
+        {canBeExecuted ? (
```

Comment:
This entire section should be re-worked. There are a lot of states here that don't make sense and the timestamp at the bottom doesn't match the state in most cases. From the design the # of states is reduced to just these:

L1 + L2 Proposals: Passed (L1), Pending (L2), Executed (once both are complete)